packages/notation/notation-tmpfiles.conf

```diff
@@ -0,0 +1,2 @@
+d /var/cache/notation - - - -
+C /etc/notation - - - - /usr/share/factory/etc/notation
```

Comment:
If the source factory is the exact same path as the target path, you can avoid it.